pallets/foreign-investments/src/impls.rs

```diff
@@ -205,23 +205,11 @@ impl<T: Config> ForeignInvestment<T::AccountId> for Pallet<T> {
 	}
 
 	fn accepted_payment_currency(investment_id: T::InvestmentId, currency: T::CurrencyId) -> bool {
-		if T::Investment::accepted_payment_currency(investment_id, currency) {
-			true
-		} else {
-			T::PoolInspect::currency_for(investment_id.of_pool())
-				.map(|pool_currency| T::Swaps::valid_pair(pool_currency, currency))
-				.unwrap_or(false)
-		}
+		T::Investment::accepted_payment_currency(investment_id, currency)
```

Comment:
**CR**

This is not correct. Now all payment currency are accepted. The above does simply check whether the currency given is the pool currency. 

cc @wischli

Reply:
That was my point in the slack conversation because trading pairs are used as payment/payout gate keepers. I thought your consensus, that it would be fine.

Reply:
I thought the same.

Reply:
I think that is fine, but we must just alwaus return true. Not checking the `T::Investments`. Sorry, for the confusion

Reply:
Ah, ok, that makes sense! Good point 🙌🏻 

Then, maybe we can remove those methods directly. WDYT?

Reply:
Removed here: [e7b70c4](https://github.com/centrifuge/centrifuge-chain/pull/1741/commits/e7b70c411d8fea83311c0d3202a8abbdd45d8e3f)